suites/apis/googleDataAccessTest.js

```diff
@@ -0,0 +1,506 @@
+  ).to.have.property('statusCode', 401);
+
+  chai.expect(User1signedUrlTest1ResFileContents,
+    "First sync: Check User1 can use signed URL to read file in test. FAILED."
+  ).to.equal(fence.props.googleBucketInfo.test.fileContents);
+
+  chai.expect(User2signedUrlTest1Res,
+    'First sync: Check that User2 could NOT get a signed URL to read file in test. FAILED.'
+  ).to.have.property('statusCode', 401);
+
+  // SECOND RUN (new authZ)
+  //  - Check Temporary Service Account Creds
+  console.log('Make assertions for user access for second run (after new usersync)');
+  console.log('First: Check temporary service account credentials');
+
+  chai.expect(user0AccessQA2Res,
+    '2nd sync: Check User0 CAN NOT access bucket for project: QA. FAILED.'
+  ).to.have.property('statusCode', 403);
+  chai.expect(user0AccessTest2Res,
+    '2nd sync: Check User0 CAN NOT access bucket for project: test. FAILED.'
+  ).to.have.property('statusCode', 403);
+
+  chai.expect(user1AccessQA2Res,
+    '2nd sync: Check User1 CAN NOT access bucket for project: QA. FAILED.'
+  ).to.have.property('statusCode', 403);
+  chai.expect(user1AccessTest2Res,
+    '2nd sync: Check User1 access bucket for project: test. FAILED.'
+  ).to.have.property('id');
+
+  chai.expect(user2AccessQA2Res,
+    '2nd sync: Check User2 access bucket for project: QA. FAILED.'
+  ).to.have.property('id');
+  chai.expect(user2AccessTest2Res,
+    '2nd sync: Check User2 access CAN NOT bucket for project: test. FAILED.'
+  ).to.have.property('statusCode', 403);
+
+  // SECOND RUN (new authZ)
+  //  - Check Signed URLs from SECOND RUN
+  console.log('Second: Check signed URLs');
+
+  chai.expect(User1signedUrlQA2Res,
+    '2nd sync: Check that User1 could NOT get a signed URL to read file in QA. FAILED.'
+  ).to.have.property('statusCode', 401);
+
+  chai.expect(User0signedUrlQA2Res,
+    '2nd sync: Check that User0 could NOT get a signed URL to read file in QA. FAILED.'
+  ).to.have.property('statusCode', 401);
+
+  chai.expect(User2signedUrlQA2ResFileContents,
+    "2nd sync: Check User2 can use signed URL to read file in QA. FAILED."
+  ).to.equal(fence.props.googleBucketInfo.QA.fileContents);
+
+  chai.expect(User0signedUrlTest2Res,
+    '2nd sync: Check that User0 could NOT get a signed URL to read file in test. FAILED.'
+  ).to.have.property('statusCode', 401);
+
+  chai.expect(User1signedUrlTest2ResFileContents,
+    "2nd sync: Check User1 can use signed URL to read file in test. FAILED."
+  ).to.equal(fence.props.googleBucketInfo.test.fileContents);
+
+  chai.expect(User2signedUrlTest2Res,
+    '2nd sync: Check that User2 could NOT get a signed URL to read file in test. FAILED.'
+  ).to.have.property('statusCode', 401);
+
+  // SECOND RUN
+  //  - Check signed URLs from FIRST RUN
+  chai.expect(User0AccessRemovedQA,
+    "Make sure signed URL from User0 CANNOT access QA data again. FAILED."
+  ).to.not.equal(fence.props.googleBucketInfo.QA.fileContents);
```

Comment:
i think `equal('some error msg')` would be safer than `not.equal('expected contents')`